src/pages/index.js

```diff
@@ -3,20 +3,68 @@ import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import "./index.css";
 
 import {
   config,
-  profileTitle,
-  profileDesc,
   inputTitle,
   inputDesc,
   editProfileButton,
-  initialCards,
   editProfileModalSelector,
+  changeProfilePicModalSelector,
+  profileImage,
 } from "../utils/constants.js";
 
+/* -------------------------------------------------------------------------- */
+/*                             API                                            */
+/* -------------------------------------------------------------------------- */
+
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "dfa65752-2d8e-401e-8af5-014c787612d4",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardList;
+let cards;
+
+const user = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
+
+api
+  .getInitialData("cards")
+  .then((data) => {
```

Comment:
Could be better: You can receive all the data using Promise.all

Reply:
This is fixed now. Thanks for the suggestion.